strategy.ini

```diff
@@ -0,0 +1,65 @@
+# some useful links:
+#  https://janelia-flyem.github.io/licenses.html
```

Comment:
based on this, shouldn't we aim for only MIT, BSD-new, Apache2.0? Do we have any weakly protective or strongly protective ones?

Reply:
yes you're right, my bad...